examples/todo/todo.py

```diff
@@ -1,29 +1,24 @@
 from flask import Flask
 from marshmallow import fields, pre_dump, Schema
 
-from flask_rebar import Rebar, errors, HeaderApiKeyAuthenticator
+from flask_rebar import Rebar, errors, HeaderApiKeyAuthenticator, Tag, \
+    SwaggerV2Generator
 from flask_rebar.validation import RequestSchema, ResponseSchema
 
-from flask_rebar import Tag,SwaggerV2Generator
 
 rebar = Rebar()
 
-# add a description of the tags we're using further down (will be output to the generated swagger)
-# a bit reduntant, given the simplicity of the code, but you get the idea :) 
+# Rebar will create a default swagger generator if none is specified. 
+# However, if you want more control over how the swagger is generated, you can 
+# provide your own. 
+# Here we've specified additional metadata for operation tags.
 generator = SwaggerV2Generator(
     tags=[
         Tag(
-            name='create',
-            description='All operations relating creation of todo items.'
+            name='todo',
+            description='All operations to managing the todo list portion of \
```

Comment:
nitpick: this is phrased a bit weird. Maybe: "Operations for managing TODO items"?

Reply:
must have been the cat walking on the keyboard ;)